apollo/resolvers/user/createUser.ts

```diff
@@ -0,0 +1,15 @@
+import bcrypt from 'bcryptjs';
+
```

Comment:
just skeleton code in this file - need to decide on lambda -> dynamo vs just postgres and typeorm